axelor-account/src/main/java/com/axelor/apps/account/service/batch/BatchCloseAnnualAccounts.java

```diff
@@ -110,6 +115,13 @@ protected void start() throws IllegalAccessException {
           batch.getId());
       incrementAnomaly();
       end = true;
+    } catch (PersistenceException e) {
+      TraceBackService.trace(
+          new AxelorException(e, TraceBackRepository.CATEGORY_CONFIGURATION_ERROR, ""),
```

Comment:
why do you create a new AxelorException with an empty message? 
`TraceBackService.trace(e,ExceptionOriginRepository.REPORTED_BALANCE,batch.getId())`  is enough